adage/adage/config.py.template

```diff
@@ -7,6 +7,10 @@
 # and replace with values tailored to your deployment.
 import random
 
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
This does not produce the right result when using `fab` to deploy. The reason is that the execution context for `fab` is not the same as the deployment target, and the location of `__file__` on the deploying server (my laptop, in this case) is not the same as where `__file__` belongs when deployed.

The specific line that's failing is `fabfile/adage_server.py`:141, but it's because we're attempting to `cd` into `CONFIG['django_dir']` before running those commands, and that construct is everywhere.

There might be another way to fix this... I'm looking into it.